src/chaplin/lib/utils.coffee

```diff
@@ -84,6 +84,12 @@ utils =
   escapeRegExp: (str) ->
     return String(str or '').replace /([.*+?^=!:${}()|[\]\/\\])/g, '\\$1'
 
+  # DOM Helpers
+  # --------------
+
+  matchesSelector: (node, selector) ->
+    matchesSelector = node.matches || node.webkitMatchesSelector || node.mozMatchesSelector || node.msMatchesSelector
+    matchesSelector.call(node, selector)
 
```

Comment:
Agreed. But maybe we could use something like https://github.com/akre54/Backbone.NativeView/blob/0.3.2/backbone.nativeview.js#L34-L48 that has older browser support and eagerly finds the selector... if we really need it